packages/core/content-manager/shared/contracts/history-versions.ts

```diff
@@ -12,7 +12,7 @@ export interface CreateHistoryVersion {
   locale: string | null;
   status: 'draft' | 'published' | 'modified' | null;
   data: Record<string, unknown>;
-  schema: Record<string, unknown>;
+  schema: Struct.SchemaAttributes;
```

Comment:
I wonder how this is different from Schema.Attributes

Reply:
I believe Schema.Attributes requires uid as a generic argument